src/test/java/steps/SearchSteps.java

```diff
@@ -0,0 +1,13 @@
+package steps;
+
+import pages.Header;
+
+public class SearchSteps {
+    private Header header = new Header();
+
+    public SearchResultSteps executeSearchByKeyword(String keyword) {
+        header.pasteToTheSearchField(keyword);
```

Comment:
I think that header is part of pages. You need to store it inside pageObjects and get access to it via pageObjects
Please remember that header is a common component for some pageObjects, please make sure you do not duplicate code